Source/Private/ASDisplayNodeCornerLayerDelegate.m

```diff
@@ -0,0 +1,27 @@
+//
+//  ASDisplayNodeCornerLayerDelegate.m
+//  Texture
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
+//  grant of patent rights can be found in the PATENTS file in the same directory.
+//
+//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
+//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+
+#import "ASDisplayNodeCornerLayerDelegate.h"
+
+@implementation ASDisplayNodeCornerLayerDelegate
+
+- (id<CAAction>)actionForLayer:(CALayer *)layer forKey:(NSString *)event
+{
+  return nil;
```

Comment:
- Should we return `(id)kCFNull` to prevent the default action, like we do for other layers? That suppresses the default action (e.g. fade in).
- Is this different than setting no delegate on the layers?
- Let's share one single instance of this rather than creating one for each layer.

Reply:
- Yes this should return `(id)kCFNull` I will change that.
- Yes setting no layer should animate the layer
- We are already sharing one instance if you look where we create the shared delegate for all of the corner layers within the code